tests/components/top_bar.test.ts

```diff
@@ -513,6 +509,29 @@ describe("TopBar component", () => {
     expect(model.getters.getCurrentContent()).toBe(content);
     app.destroy();
   });
+
+  test.each([
+    ["Horizontal align", ".o-dropdown-content"],
+    ["Borders", ".o-dropdown-content"],
+    ["Font Size", ".o-dropdown-content"],
+    ["Fill Color", ".o-color-picker"],
+    ["Text Color", ".o-color-picker"],
+    ["More formats", ".o-dropdown-content"],
+  ])(
+    "Clicking inside a dropdown '%s' don't close the dropdown",
```

Comment:
This test is a bit weird because the "normal" behavior is... *to close the dropdown*.
```suggestion
    "Clicking a static element inside a dropdown '%s' don't close the dropdown",
```